### ISSUE TITLE
null backend is missing capture support

### ISSUE THREAD

**Issue by t6** [kcat/openal-soft#206]
Not all backends have capture support and applications crash when trying to enumerate available capture devices and when there is no usable backend available for that. You can easily trigger this by running `openal-info`:
```
ALSOFT_DRIVERS=null openal-info 
Available playback devices:
    No Output
Available capture devices:
Segmentation fault
```
Tested on FreeBSD and Void Linux with 1.18.2.

**Comment by kcat** [comment]
The crash should already be fixed in the Git version, can you check that? The null backend still doesn't support capture though, it wouldn't produce any usable samples if it did.

**Comment by t6** [comment]
Yes, it seems to work with the Git version. Do you think that the fix is easily backportable to 1.18.2?
